ColorPalette.js

```diff
@@ -84,7 +83,7 @@ define([
 			// Creates <img> nodes in each cell of the template.
 			this._preparePalette(
 				this._palettes[this.palette],
-				i18n.getLocalization("dojo", "colors", this.lang));
```

Comment:
And then prior to the call to _preparePalette:

var _i18n;
if(has("csp-restrictions")){
	_i18n = i18n.getLocalization("dojo", "colors", this.lang));
}else{
	_i18n = i18n;
}